frinx-uniconfig/translation-units-docs/operational-datasets/platform.md

```diff
@@ -83,11 +83,83 @@ frinx-openconfig-platform:components
+                    },
+                    "frinx-arris-platform-extension:transceiver: {
+                        "serial-number": "<transceiver_serial_number>",
+                        "temperature": "<transceiver_temperature>",
+                        "vendor": "<transceiver_vendor>",
+                        "revision": "<transceiver_revision>",
+                        "date-code": "<transceiver_date_code>",
+                        "ch-0-tx-bias": "<transceiver_ch_0_tx_bias>",
+                        "t-type": "<transceiver_t_type>",
+                        "voltage": "<transceiver_voltage>",
+                        "speed": "<transceiver_speed>",
+                        "part-number": "<transceiver_part_number>",
+                        "type": "<transceiver_type>",
+                        "ch-0-tx-power": "<transceiver_ch0_tx_power>",
+                        "ch-0-rx-power": "<transceiver_ch0_rx_power>",
+                    },
+                    "frinx-arris-platform-extension:chassis": {
+                        "model-version": "<chassis_model_version>",
+                        "serial-number": "<chassis_serial_number>",
+                        "model-name": "<chassis_model_name>"
+                    },
+                    frinx-arris-platform-extension:version-detail": {
+                        "serial-number": "<version_serial_number>",
+                        "cpu-speed": "<version_cpu_speed>",
+                        "active-sw": "<version_active_sw>",
+                        "active-patch": "<version_active_patch>",
+                        "pic-serial-number": "<version_pic_serial_number>",
+                        "ram-size": "<version_ram_size>",
+                        "model-name": "<version_model_name>",
+                        "nor-flash-size": "<version_nor_flash_size>",
+                        "nand-flash-size": "<version_nand_flash_size>",
+                        "model-version": "<version_model_version>",
+                        "reason-last-booted": "<version_reason_last_booted>",
+                        "firmware-version": "<version_firmware_version>",
+                        "bus-speed": "<version_bus_speed>",
+                        "uptime": "<version_uptime>",
+                        "type": "<version_type>",
+                        "pic-model-name": "<version_pic_model_name>",
+                        "pic-model-version": "<version_pic_model_version>"
                     }
                 },
                 "config": {
                     "name": "<component_key>"
                 }
+            },
+            {
+                "name": "OS",
+                "state": {
+                    "name": "OS",
+                    "frinx-arris-platform-extension:license": {
+                        "system-lld-us-asf: <license_lld_us_asf>,    // true or false
+                        "chassis-serial-number": "<license_chassis_serial_number>,
```

Comment:
there is typo: "<license_chassis_serial_number>
use "<string>" or <other>

Reply:
OK